app/routes/events/components/EventDetail/index.tsx

```diff
@@ -227,7 +227,7 @@ export default class EventDetail extends Component<Props, State> {
 
     // The UserGrid is expanded when there's less than 5 minutes till activation
     const minUserGridRows = currentMoment.isAfter(
-      activationTimeMoment.subtract(5, 'minutes')
+      moment(activationTimeMoment).subtract(5, 'minutes')
```

Comment:
This is just `moment(moment(event.activationTime))`? How does that do anything?

Reply:
It makes sure the value of activationTimeMoment isn't changed

Reply:
And how does this have anything to do with the displaytime? This is for expanding the user grid 🤔

Reply:
`activationTimeMoment.subtract(5, 'minutes')` mutates `activationTimeMoment` in-place, which is used elsewhere.

So this ensures that we don't edit that moment object but create a new one for the subtracted time.

Reply:
ah ffs lol okay of course it's in-place